app/assets/stylesheets/components/_contents-list-with-body.scss

```diff
@@ -36,6 +36,7 @@
   }
 
   .app-c-back-to-top {
+    color: $govuk-blue;
```

Comment:
Should this use $link-colour?
Does this respond correctly to focus state?

Reply:
I've amended to use `$link-colour`, `$link-active-colour` and `$link-hover-colour`